### PR TITLE
Sa 2825 sentinel one install scripts command gallery

### PR DESCRIPTION
**Pull request by kmaranionjc**
## Issues
* [SA-2825](https://jumpcloud.atlassian.net/browse/sa-2825) - SA-2825-SentinelOne-Install-Scripts-Command-Gallery

## What does this solve?
Ability to install Sentinel One agent using Commands

## Is there anything particularly tricky?
* Linux and Windows S1 install does not work for arm based machines (M1 MacBooks)
* Make sure to uninstall the S1 agent after a successful install due to it that it can lock down your VM network and JC agent. [Uninstall instructions](https://www.cybervigilance.uk/post/sentinelone-uninstalling-agent)

## How should this be tested?

* Mac
1. Add the Sentinel One Agent Permissions Policy to your Mac Device
2. Create a .sh file, change the sentinelToken and DownloadURL variables and run this [script](https://github.com/TheJumpCloud/support/blob/SA-2825-SentinelOne-Install-Scripts-Command-Gallery/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20Sentinel%20One%20Agent.md)
3. Sentinel One agent should pop up on the top right icons and it shouldn't ask for any security permissions
4. Uninstall using this command: sudo sentinelctl uninstall --local
- Paraphrase is located in the S1 console agent actions

* Windows
1. Edit siteToken and installerURL variables then run the [powershell script ](https://github.com/TheJumpCloud/support/blob/SA-2825-SentinelOne-Install-Scripts-Command-Gallery/PowerShell/JumpCloud%20Commands%20Gallery/Windows%20Commands/Windows%20-%20Install%20Sentine%20One%20Agent.md)
2. After successful install, uninstall the agent

* Linux
1. Edit token and DownloadURL then run the [script](https://github.com/TheJumpCloud/support/blob/SA-2825-SentinelOne-Install-Scripts-Command-Gallery/PowerShell/JumpCloud%20Commands%20Gallery/Linux%20Commands/Linux%20-%20Install%20Sentinel%20One.md) 
2. After successful install, uninstall the agent

### CODE REVIEW

File: PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install Sentinel One Agent.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Write-Host "Installing Sentinel One Agent now, this may take a few minutes."
+try {
+    ."$installerTempLocation" --dont_fail_on_config_preserving_failures -t $siteToken
+}
+catch {
+    Write-Error "Failed to run Sentinel One Agent installer."
+    exit 1
+}
+Write-Host "Sentinel One Agent installer returned $($installerProcess.ExitCode)."
```
Comment (jworkmanjc) [inline PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install Sentinel One Agent.md:43]: This looks good but when I ran this I did see the command results return the following:

Screen Shot 2022-09-20 at 2 36 28 PM

The text of the exit code 0 is not returned here

**Review by jworkmanjc — APPROVED** [review]
I'm good with these script examples! Thanks for adding the bit for Linux!

File: PowerShell/JumpCloud Commands Gallery/Linux Commands/Linux - Install Sentinel One.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+linux
+
+#### Command
+
+```
+###################################################
+sentinelToken=YOURTOKEN
+filename=sentineloneagent.deb
+DownloadURL=https://path/to/url.deb
```
[the remark this answers is not in the document]
Comment (jworkmanjc) [reply PowerShell/JumpCloud Commands Gallery/Linux Commands/Linux - Install Sentinel One.md]: We are now checking for the agent binary marking this resolved!

**Review by gweinjc — APPROVED** [review]
Looks good!

File: PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install Sentinel One Agent.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Windows - Install Sentinel One Agent| v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+$siteToken="SITE TOKEN"
```
Comment (jworkmanjc) [inline PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install Sentinel One Agent.md:12]:
```suggestion
$siteToken="YOURTOKEN"
```

**Pull request merged**